New package: PCMan-BBS.PCMan version 9.5.0 Beta 3

sbvcid opened the pull request:
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/307063)

sbvcid committed:
New package: PCMan-BBS.PCMan version 9.5.0 Beta 3

stephengillie reviewed (changes requested):
Hi @sbvcid,

Manual Validation ended with:
<img width="873" height="474" alt="image" src="https://github.com/user-attachments/assets/0e079833-6855-48e6-9aa3-eb005a59c42a" />

The isntaller launched the installer for its dependency, which is waiting for user interaction. Usually this would be a blocker. But there's a chance that preinstalling the dependency might help the installer to skip it, and complete the install unattended.

sbvcid committed:
Update manifests/p/PCMan-BBS/PCMan/9.5.0 Beta 3/PCMan-BBS.PCMan.installer.yaml

Apply suggested dependency update from @stephengillie
Added Dependencies section to specify Microsoft.VCRedist.2015+.x64

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

sbvcid commented on manifests/p/PCMan-BBS/PCMan/9.5.0 Beta 3/PCMan-BBS.PCMan.installer.yaml:
```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: PCMan-BBS.PCMan
+PackageVersion: 9.5.0 Beta 3
+InstallerType: nullsoft
+Installers:
```
Hi @stephengillie,
Thanks for the feedback! The screenshot shows a VC++ 2017 Redistributable installer.
Should we use Microsoft.VCRedist.2017+.x64 as the dependency?

stephengillie commented:
Automatic Validation ended with: 
> 
> MSI (s) (B8:B4) [22:11:08:765]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (B8:B4) [22:11:08:765]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (B8:B4) [22:11:08:765]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (B8:B4) [22:11:08:765]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (B8:B4) [22:11:08:765]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.
> Property(S): OutOfDiskSpace = 0
> Property(S): OutOfNoRbDiskSpace = 0
> Property(S): PrimaryVolumeSpaceAvailable = 0
> Property(S): PrimaryVolumeSpaceRequired = 0
> Property(S): PrimaryVolumeSpaceRemaining = 0
> MSI (s) (B8:50) [22:11:09:656]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (B8:50) [22:11:09:656]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (B8:50) [22:11:09:656]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (B8:50) [22:11:09:656]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (B8:50) [22:11:09:656]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.

(Automated response - build 1142.)

Trenly commented:
[Policy] Needs Author Feedback

sbvcid committed:
Reorganize dependencies in installer YAML

sbvcid committed:
Update PCMan installer YAML with new fields

Added installer switches and scope for machine installation.
